src/test/java/org/springframework/samples/petclinic/integration/Browser.java

```diff
@@ -14,6 +14,10 @@ public class Browser {
 
 	public final static WebDriver webDriver = createDriver();
 
+	public static void browserClose() {
+		webDriver.close();
+	}
+
```

Comment:
Try to add this code to the `createDriver()` method and run the `AddOwners.feature` tests:
```java
Map<String, ? extends Serializable> slowConnectionSettings = Map.of(
	"offline", false,
	"latency", 5,
	"download_throughput", 1024 * 20,
	"upload_throughput", 1024 * 20
);
CommandExecutor executor = chromeDriver.getCommandExecutor();
try {
	Response response = executor.execute(
		new Command(chromeDriver.getSessionId(),
			"setNetworkConditions",
			Map.of("network_conditions", slowConnectionSettings)));
} catch (IOException e) {
	throw new RuntimeException(e);
}
```

You'd also need to change the Java version in `pom.xml` from `1.8` to `17` (because I wanted to make the example code short):
```xml
<java.version>17</java.version>
```

Some of the tests are starting to fail. Such hiccup is a normal situation on CI servers. How can we improve our tests to avoid this flakiness?